base/applications/sndvol32/misc.c

```diff
@@ -158,6 +158,118 @@ CloseAppConfig(VOID)
     }
 }
 
+BOOL
+LoadXYCoordWnd(IN PREFERENCES_CONTEXT PrefContext)
+{
+    HKEY hKey;
+    LONG lResult;
+    TCHAR DeviceMixerSettings[356];
+    DWORD dwData;
+    DWORD cbData = sizeof(dwData);
+
+    /* Append the registry key path and device name key into one single string */
+    StringCchPrintf(DeviceMixerSettings, _countof(DeviceMixerSettings), TEXT("%s\\%s"), AppRegSettings, PrefContext.DeviceName);
+
+    lResult = RegOpenKeyEx(HKEY_CURRENT_USER,
+                           DeviceMixerSettings,
+                           0,
+                           KEY_READ,
+                           &hKey);
+    if (lResult != ERROR_SUCCESS)
+    {
+        return FALSE;
+    }
+
+    lResult = RegQueryValueEx(hKey,
+                              TEXT("X"),
+                              0,
+                              0,
+                              (LPBYTE)&dwData,
+                              &cbData);
+
+    if (lResult != ERROR_SUCCESS)
+    {
+        RegCloseKey(hKey);
+        return FALSE;
+    }
+
+    /* Cache the X coordinate point */
+    PrefContext.MixerWindow->WndPosX = dwData;
+
+    lResult = RegQueryValueEx(hKey,
+                              TEXT("Y"),
+                              0,
+                              0,
+                              (LPBYTE)&dwData,
+                              &cbData);
+
+    if (lResult != ERROR_SUCCESS)
+    {
+        RegCloseKey(hKey);
+        return FALSE;
+    }
+
+    /* Cache the Y coordinate point */
+    PrefContext.MixerWindow->WndPosY = dwData;
+    
+    RegCloseKey(hKey);
+    return TRUE;
+}
+
+BOOL
+SaveXYCoordWnd(IN HWND hWnd,
+               IN PREFERENCES_CONTEXT PrefContext)
+{
+    HKEY hKey;
+    LONG lResult;
+    TCHAR DeviceMixerSettings[356];
```

Comment:
Why using TCHAR? Is WCHAR the better?

Reply:
Because most of the application's code is Unicode/ANSI agnostic, that is, the compiler may compile it as a Unicode software if Unicode is defined and viceversa. In no way I'm going to change this to a `WCHAR`.

Reply:
OK. Please ignore my W-related reviews.